cogs/events.py

```diff
@@ -1,27 +1,89 @@
 import traceback
+import textwrap
+from .utils import constants
 
 import discord
 from discord.ext import commands
 
+PREFIX = constants.COMMAND_PREFIX
+
 
 class Custom(commands.Cog):
 
     def __init__(self, client):
         self.client = client
+        self.dev_mode = constants.DEV_MODE
 
     @commands.Cog.listener()
     async def on_message(self, message):
         if message.author.id == self.client.user.id:
             return
 
-        if message.content.lower().startswith("help"):
-            await message.channel.send("Hey! Why don't you run the help command with `>>help`")
+        if message.author.bot:  # make sure the bot doesn't respond to other bots
+            return
 
         await self.client.process_commands(message)
 
+    @commands.Cog.listener()
+    async def on_guild_join(self, guild: discord.Guild):
+
+        hw = self.client.get_user(715545167649570977)
+        logchannel = self.client.get_channel(704197974577643550)
```

Comment:
These IDs shouldn't be hard coded, maybe they can also be in `constants.py`.
Same goes for line 77